exercises/practice/largest-series-product/src/test/scala/SeriesTest.scala

```diff
@@ -3,7 +3,7 @@ import org.scalatest.matchers.should.Matchers
 
 
 /** @version 1.1.0 */
-class LargestSeriesProductTest extends AnyFunSuite with Matchers {
+class SeriesTest extends AnyFunSuite with Matchers {
```

Comment:
Maybe in this case, the test _file_ (not the class) should be updated? This might be confusing with there also being a Series exercise.

Reply:
I wasn't aware of that other exercise.
Renamed!